pkg/controller/utils/utils.go

```diff
@@ -430,6 +430,7 @@ func WaitToAddResourceWatch(controller controller.Controller, client kubernetes.
 	duration := 1 * time.Second
 	ticker := time.NewTicker(duration)
 	defer ticker.Stop()
+	log.Info("handlng watch of %s resource", obj.GetObjectKind().GroupVersionKind().Kind)
```

Comment:
Maybe "waiting for %s API to become available for watching" ?

Reply:
Would it be worth adding the `kind` to a context logger for this loop so it's present on subsequent logs as well?